app/src/main/java/com/burhanrashid52/photoediting/tools/ToolType.java

```diff
@@ -7,6 +7,8 @@
  */
 public enum ToolType {
     BRUSH,
+    OVAL,
```

Comment:
Can we use just `Shape` here since this is part of public it will break more often if we add/remove new Shapes

Reply:
I guess we can ventilate Shape choice directly from the Builder, so why not.

Reply:
For now we just use `ToolType.Shape` to indicate a Shape not the shape details. I am fine with another enum as well.

Reply:
The thing is this enum is tightly coupled with the UI, as it is aligned with the tool selector. I will propose something simple to select the shape in the UI, but it is up to each developer to make its own implementation.